release-plz.toml

```diff
@@ -0,0 +1,24 @@
+[workspace]
+changelog_update = false # disable changelog updates
+dependencies_update = true # update dependencies with `cargo update`
+git_release_enable = true # enable GitHub/Gitea releases
```

Comment:
We should add `git_release_draft = true` too so we can edit the release notes before publishing.